packages/elements/src/components/metrics-table/metrics-table.component.ts

```diff
@@ -40,65 +40,95 @@ export class MutationTestReportTestMetricsTable<TFile, TMetric> extends LitEleme
     low: 60,
   };
 
+  public static styles = [tailwind];
+
+  private hasMultipleColspan = false;
+
+  public override willUpdate(changedProperties: PropertyValues) {
+    if (changedProperties.has('columns')) {
+      this.hasMultipleColspan = this.columns.some((column) => column.category === 'percentage');
```

Comment:
This feels like magic. Can we make it a getter, or calculate this value when we need it?

Reply:
I had it as a getter first, but that meant that the value would be recalculated for each table cell. The function is small, but for a large enough file list it adds up. This way at least it's only calculated once. I still don't like how magic it feels (percentage category means multiple colspan???) but I wouldn't know of a cleaner way